src/components/MainContent/index.js

```diff
@@ -23,11 +26,10 @@ function MainContent() {
     };
 
     const onFavoriteClick = () => {
-        // on click look up local storage item
-        // add new coin id to local storage item
-        const newFavorites = [...favorites, data.id];
-        localStorage.setItem("favoriteCoinIds", newFavorites);
-        dispatch(updateFavorites(newFavorites));
+        // dispatch addNewFavorite with argument of coin id saved in state in this component - powering the value of the input
```

Comment:
This sort of comment/documentation is not needed. The reason being, you've named your function/redux action semantically so it's self documenting. Which is exactly how you should aim to have all of your functions and variables named.